### DISCUSSION TITLE
animation with background and objects.

### DISCUSSION THREAD

**Discussion by CaiYitao** [touying-typ/touying#46] [Ideas]
Can Touying du background and objects animation, for example like the following, at the beginning, it is all white background, and then the black background will be pulled up from the bottom, and then the black background will rise to cover the whole slide, together with the objects.

![AF48DBD51AC75069C9D50FEB08A6B479](https://github.com/touying-typ/touying/assets/65641330/707976db-d9c9-48cd-bd56-ba70cad8cffa)

![9601DB6ECD8536814711C71430431E13](https://github.com/touying-typ/touying/assets/65641330/835f4ee6-8ddb-4d32-9087-d7d15a8bb223)

**Comment by OrangeX4** [comment]
No at the moment, Touying doesn't support any transition animations at the moment, only static pages
